dhis-2/pom.xml

```diff
@@ -2103,72 +2061,211 @@
         </exclusions>
       </dependency>
 
-      <!-- https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on -->
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcprov-jdk15on</artifactId>
-        <version>1.66</version>
+        <version>${bcprov-jdk15on.version}</version>
       </dependency>
 
-
-      <!-- https://mvnrepository.com/artifact/xerces/xercesImpl -->
       <dependency>
         <groupId>xerces</groupId>
         <artifactId>xercesImpl</artifactId>
-        <version>2.12.0</version>
+        <version>${xercesImpl.version}</version>
       </dependency>
 
       <dependency>
         <groupId>org.mapstruct</groupId>
         <artifactId>mapstruct</artifactId>
         <version>${mapstruct.version}</version>
       </dependency>
-
     </dependencies>
 
 
   </dependencyManagement>
 
   <properties>
```

Comment:
I'm surprised to find the properties section this far down. Isn't this the section you want to see first?

Reply:
I agree, I can move it to the top and see what reactions it will get :)